css/style.css

```diff
@@ -1,6 +1,163 @@
 /* General page styles */
 
-/* We should use a mobile first approach having out breack point at 768px */
+/* We should use a mobile first approach having out brake point at 768px */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.container {
+	position: relative;
+	margin: auto;
+	max-width: 960px;
+	height: 100%;
+}
+
+.main-nav {
+	position: fixed;
+	top: 0;
+	z-index: 2;
+	width: 100%;
+	height: 75px;
+	overflow-x: hidden;
+	transition: 0.5s;
+}
+
+.colored__main-nav {
+	background-color: rgba(0, 60, 143, 0.8);
+}
+
+.transparent__main-nav {
+	background-color: rgba(0, 60, 143, 0);
+}
```

Comment:
According to BEM, this should be a modifier. So, it should be named like main-nav--colored and main-nav--transparent.
Please check and fix.

Reply:
True, fixed.